### ISSUE TITLE
Fallback definition and authoring guidance is missing

### ISSUE THREAD

**Issue by gautierchomel** [mattgarrish/epub-revision#2773]
### Describe the problem

The [Fixed layout accessibility guidance for reading systems](https://w3c.github.io/epub-specs/wg-notes/fxl-a11y/#rs-a11y) section has a dedicated [5.3.9Support fallbacks](https://w3c.github.io/epub-specs/wg-notes/fxl-a11y/#rs-a11y-requirements-fallbacks) subsection but the Fallback concept is only mentioned once in the authoring part of the document and once in the technique document.

For content creators, that can be confusing as a whole possible authoring part is not explained or addressed. 

### Describe the fix or new feature you propose

Add a section between 2 and 3 to describe fallback and how it can be implemented or not in a fixed layout context. 

Alternatively, a subsection in one of sections 1 or 2 could help.

**Comment by iherman** [comment]
This was discussed during the [pmwg meeting on 28 August 2025](https://w3c.github.io/pm-wg/minutes/2025-08-28.html#569b).

<details><summary><i>View the transcript</i></summary>

<h3>Fallbacks</h3>
<p><cite>&lt;shiestyle&gt;</cite> <a href="https://github.com/w3c/epub-specs/issues/2773" rel="noopener noreferrer">w3c/<wbr>epub-specs#2773</a></p>
<p><cite>gautierchomel:</cite> The question from FL a11y guidance doc (a hybrid doc), on reviewing it the RS part has some stuff about supporting fallbacks. It says "in cases where it is supplied, RS should support"</p>
<p><cite>gautierchomel:</cite> but we are missing a section on what can be done or not with them. The only guidance is in the main spec, which does not say what you cannot do<br>
<span>… there was some work done on auto-reflow on FL, with intent to use HTML fallback on an HTMLresource</span><br>
<span>… We realized it wasn't the right way, but if that was attempted then it means the spec isn't clear enough</span><br>
<span>… For example, can I have an html fallback for svg? Spec doesn't say no</span><br>
<span>… to me this is a question to address in the FL doc, since it is specific to reflow documents in FL docs</span></p>
<p><cite>gautierchomel:</cite> but group decided not to address it there and do it here, though I disagree with that and think it should be in the FL doc</p>
<p><cite>duga:</cite> why do you say html fallbacks of html aren't the right way to do it?<br>
<span> …did it break reading systems and not work in practice?</span></p>
<p><cite>gautierchomel:</cite> the reading system already accepts fixed layout. I want you to use a fall back, we were asked to add a button to select the fall back. We can't do that.<br>
<span>… in thorium we will offer a fall back view, but we won't be allowed to switch between them</span></p>
<p><cite>duga:</cite> I don't know what is right for your reader, when we created the fall back spec we decided it was legal to have a same type fallback ie html with an html fallback<br>
<span>… we didn't specify how a reading system can determine which file is the fall back</span><br>
<span>… I don't think we should limit the fallback and say you can't fall back to the same filetype</span></p>
<p><cite>mgarrish:</cite> I agree that we could fall back to a different file type. This is an interesting case, because fallbacks weren't made with fixed layout in mind<br>
<span>… how do we indicate which html is fixed layout and which is reflowable?</span></p>
<p><cite>wendyreid:</cite> the other thing making this complicated is that this use case is maybe out of step with the intention of fallbacks.<br>
<span>… "the reading system can't render the original" so it has a file it can render</span><br>
<span>… this particular usage is not that the reading system can't render the original, but that we want to give the user choices</span><br>
<span>… there should be a different mechanism for that. There could be other alternative cases</span></p>
<p><cite>gautierchomel:</cite> we should not address the reading system giving a choice to the reader, that is up to the reading system<br>
<span>… some reading systems don't display anything when they encounter a fixed layout</span><br>
<span>… still we should know what we should and shouldn't do when an rs cannot display an fxl</span><br>
<span>… and does this go in the fxl document or the main document</span></p>
<p><cite>Hadrien:</cite> I have seen things like this before,<br>
<span>… Barnes and Noble wanted to keep the print like experience, but also offer a reflowable</span><br>
<span>… but there isn't a one to one mapping between the fxl and the reflowable assets</span><br>
<span>… B&amp;N chose multiple renditions with a document that could map an area in fxl to the reflowable</span><br>
<span>… Hachette creates multiple rendition epubs without mapping</span><br>
<span>… fallback will not work well because of the mapping issue</span><br>
<span>… we probably need something more like an alternate</span><br>
<span>… there is a similar proposal in the comics task force</span><br>
<span>… it would be easier for some publishers to have an easy way to map the html to a resource</span></p>
<p><cite>gautierchomel:</cite> let's separate the discussion about fallbacks and alternate renderings<br>
<span>… still the issue about what should I do in an fxl file that can't be rendered</span></p>
<p><cite>wendyreid:</cite> the challenge is that an fxl file can't be shown or renders badly<br>
<span>… this is like the audio only readers, the problem isn't fixed layout, but that the file doesn't follow semantic html</span><br>
<span>… so the guidance we have in place would still solve the issue</span></p>
<p><cite>Hadrien:</cite> we are working on Thorium mobile, we already support the ability to listen to an fxl<br>
<span>… with a well structured fxl you can have a decent voice experience. But this isn't always the case</span><br>
<span>… we will try to process and find sentences and groups of words</span><br>
<span>… it won't work if there is no text or text is out of order</span></p>
<p><cite>wendyreid:</cite> our emphasis should remain on well structured content<br>
<span>… but we should discuss alternatives</span></p>
<p><cite>ivan:</cite> we took multiple rendition out of 3.3 because there was no implementation for this<br>
<span>… is that still true, or should we keep in the past</span></p>
<p><cite>Hadrien:</cite> there is implementation within propriety specialized reading systems<br>
<span>… not usually as an interoperable format</span><br>
<span>… does anyone know if B&amp;N still uses this?</span></p>
<p><cite>ivan:</cite> does vital source use it?<br>
<span>… if not, we should keep multiple rendition in the past</span></p>
<p><cite>sueneu:</cite> what is the next step? Should we make recommendations for creators?</p>
<p><cite>wendyreid:</cite> This is for the fxl accessibility task force, I don't recommend using html fallbacks because it isn't proven.</p>
<p><cite>George:</cite> the user who doesn't want that experience may want an alternative<br>
<span>… can we create this?</span></p>
<p><cite>duga:</cite> we don't have to solve the world's problems<br>
<span>… it sounds like the issue is we want to use fall backs</span><br>
<span>… we tried this but it doesn't work.</span><br>
<span>… for fixed layout books, we might have multiple documents point to a single html</span><br>
<span>… do we simply need to say that fall backs don't work in this case?</span></p>
<p><cite>gautierchomel:</cite> one problem at a time. We just need to put a note in the fixl document<br>
<span>… maybe in fxl specs</span><br>
<span>… in the future we should let content creators know how to restyle this html</span><br>
<span>… in the future multiple rendition might come back</span></p>
<p><cite>Hadrien:</cite> what we want to do in Thorium is the style selection you can do in a browser<br>
<span>… and use no original styling</span><br>
<span>… I am doubtful that people will be excited about this and use it again</span></p>
<p><cite>Hadrien:</cite> what we want to do in thorium web is the equivalent of the reading mode in the browser, so there's no way to style this form a content creator perspective. Multiple rendition have not been used, we should not want to try again.<br>
<span>… I would be happy to point people to what already exists</span></p>
<p><cite>gautierchomel:</cite> does anyone know of a specification on reading mode provided by a browser</p>
<p><cite>ivan:</cite> I haven't seen that</p>
<p><cite>ack:</cite> CharlesL</p>
<p><cite>CharlesL:</cite> back in the day there was a reader who could show you a scan or a reflowable ocr rendering</p>
<p><cite>DaleRogers:</cite> I created an epub, fxl, a short comic, with a reflowable introduction and backmatter that was reflowable<br>
<span>… according to the spec I can do this, it worked in Apple Books</span><br>
<span>… Amazon wouldn't take it. I chatted with someone who looked at his epub</span><br>
<span>… they suggested uploading it as a comic book</span><br>
<span>… and not allow the front matter and back matter to be reflowable</span><br>
<span>… their creator tool requires loading in PDFS that can't be made accessible</span><br>
<span>… there are some platforms on which this will be accessible and some where it isn't</span><br>
<span>… even though the spec says I should be able to do this</span><br>
<span>… it takes me aback that I can't always use features listed in the spec</span></p>
<p><cite>shiestyle:</cite> in japan, if we want to mix fxl and reflow, we generate a reflow with some resources fixed.</p>
<p><cite>sueneu:</cite> so two different discussion. One to go to the fxl group and the other to keep in mind for the future.</p>
<p><cite>duga:</cite> my understanding is that we want to explicitly say what won't work.</p>
<p><cite>&lt;duga&gt;</cite> +1 to Hadrien</p>
<p><cite>Hadrien:</cite> probably, a problem is that many properties are spine based while they would give better benefit in the manifest</p>

<hr /></details>
